src/pages/billing/Project.js

```diff
@@ -716,8 +754,18 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
         setCostPerWorkspace(costPerWorkspace)
         setUpdatingProjectCost(false)
       }
+    }
+    Utils.withBusyState(setUpdating)
```

Comment:
I changed to `Utils.withBusyState()`, but when I use this, I noticed that the loading icon/page doesn’t show up even when the page hasn’t fully loaded yet-- I’m not sure if I'm calling Utils.withBusyState() in the right place.